core/jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -27,7 +27,9 @@ import { environment as env_oss } from './../../../../environments/environment.o
 export class CreateServiceComponent implements OnInit {
 
   @Output() onClose:EventEmitter<boolean> = new EventEmitter<boolean>();
-
+  sqsStreamString:string = "arn:aws:sqs:us-west-2:" + env_oss.aws.account_number + ":stream/";
```

Comment:
can this be config driven. also do we need to hardcode the region?
why all the the arns have stream at the end? is that how the arn is defined?

Reply:
moved to config,
updated arns